src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -246,6 +251,33 @@ public RevisionsCollectionConfiguration GetRevisionsConfiguration(string collect
                         }
                     }
 
+                    if (fromReplication)
+                    {
+                        void PutFromRevision() => _documentsStorage.Put(context, id, null, document, lastModifiedTicks, changeVector, 
```

Comment:
Let us use the full method syntax here, it took me a while to figure out that you are defining a local method using expression syntax.